README.md

```diff
@@ -23,23 +25,23 @@ Codewind for VS Code provides tools to help build cloud-native, containerized ap
 - Toggle the project auto build setting and manually initiate project builds.
 - Open your application or the application monitor in a browser.
 
-For more information, see the [Codewind website](https://www.eclipse.org/codewind/).
-
-## Installing Codewind
-You can install Codewind locally in VS Code. For more information about installing Codewind, see [Installing Codewind for VS Code](https://www.eclipse.org/codewind/mdtvscinstallinfo.html).
+## Getting Started with Codewind
+For more information about getting started with Codewind, see [Getting Started with Codewind for VS Code](https://www.eclipse.org/codewind/mdt-vsc-getting-started.html).
 
-Prerequisites
+### Prerequisites
 - Install [VS Code](https://code.visualstudio.com/download) 1.28.0 or later.
 - Install Docker 17.06 or later.
 - If you use Linux, you also need to install Docker Compose.
 
-Complete the installation:
+### Complete the installation:
+
+For more information, see [Installing Codewind for VS Code](https://www.eclipse.org/codewind/mdt-vsc-installinfo.html).
+
 1. Find Codewind for VS Code in the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind) or by searching for `Codewind` in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
-2. Open the **Codewind** view in the Explorer view group or enter `Focus on Codewind View` into the [**Command Palette**](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette). If you do not see the Codewind view in either the Explorer view or the **Command Palette**, the extension did not install correctly.
+2. Open the **Codewind** view in the Explorer view group or enter `Focus on Codewind View` into the [**Command Palette**](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette). If you do not see Codewind in either the Explorer view or the Command Palette, the extension did not install correctly.
 3. Choose **Install** when prompted to complete the installation of additional Docker images that Codewind requires to run. The installation may take a few minutes to complete.
-
-## Using Codewind for VS Code
-To see the actions available, open the [**Command Palette**](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) and type `Codewind`.</br>
+4. Start by creating or adding a project. You can add a project by using either  the `Create Project` or `Add Existing Project` command. The new project appears in the Codewind view.
+5. To see all actions available, open the Command Palette and type `Codewind`. Commands can also be accessed through buttons and right-click menus in the Codewind view.
```

Comment:
Remove passive voice in the second sentence by changing it to "You can also access commands through..."